parse/EnumPythonParser.h

```diff
@@ -11,7 +11,7 @@ template<typename E>
 struct enum_wrapper {
```

Comment:
is this struct really necessary? It's just wrapping a single enum value...

Reply:
Yes, raw values got issues with using in Python so I've added wrapper around them.

Reply:
seems strange... if [it works for `int`](https://github.com/freeorion/freeorion/commit/e0ac49cce2825b373661ccc281bccc1fc42523db#diff-607966e20c51bd22fc996098ace39cce0265eb0ece7c44349f7e2ce4bfccc498L292) why wouldn't it work for enums...?

Reply:
`2: [2023-03-15 10:48:11.702553] [0x00007faa06ba4780] [info]    TypeError: No registered converter was able to produce a C++ rvalue of type PlanetEnvironment from this Python object of type __PlanetEnvironment`

https://github.com/freeorion/freeorion/actions/runs/4425439377/jobs/7760495106#step:13:127

Reply:
It should be solved bu declaring `class_` for them.